cpu/native/include/periph_cpu.h

```diff
@@ -96,6 +96,52 @@ typedef enum {
 } gpio_flank_t;
 
 /** @} */
+#else
```

Comment:
```suggestion
#else  /* MODULE_PERIPH_GPIO_LINUX | DOXYGEN */
```

but shouldn't there be some GPIO_MOCK module to activate this?

Reply:
done (see next comment)